pyvistaqt/window.py

```diff
@@ -1,21 +1,27 @@
+"""
+TODO: Add docstring
+"""
 from PyQt5 import QtCore
 from PyQt5.QtCore import pyqtSignal
 from PyQt5.QtWidgets import QMainWindow
 
 
 class MainWindow(QMainWindow):
-    """Convenience MainWindow that manages the application."""
+    """
+    Convenience MainWindow that manages the application.
+    """
 
     signal_close = pyqtSignal()
     signal_gesture = pyqtSignal(QtCore.QEvent)
 
     def event(self, event):
+        """TODO: Add docstring"""
         if event.type() == QtCore.QEvent.Gesture:
             self.signal_gesture.emit(event)
             return True
         return super().event(event)
 
-    def closeEvent(self, event):
+    def close_event(self, event):
```

Comment:
Won't a lot of these actually break the code? `closeEvent` is a Qt event/method so this will no longer be called

Reply:
@larsoner Thanks for your comment. I will revert this and enable pylint here.
```suggestion
    def closeEvent(self, event):
```